tree/tree/src/TTree.cxx

```diff
@@ -100,9 +100,9 @@ It is strongly recommended to persistify those as objects rather than lists of l
    - `I` : a 32 bit signed integer (`Int_t`)
    - `i` : a 32 bit unsigned integer (`UInt_t`)
    - `F` : a 32 bit floating point (`Float_t`)
-   - `f` : a 21 bit floating point with truncated mantissa (`Float16_t`): 1 for the sign, 8 for the exponent and 12 for the mantissa. Can be customized with suffix `[min,max(,nbits)]`.
+   - `f` : a 24 bit floating point with truncated mantissa (`Float16_t`, stored as 3 bytes, occupies 4 bytes in memory): By default, in disk, only 21 bits are used: 1 for the sign, 8 for the exponent and 12 for the mantissa. Can be customized with suffix `[min,max(,nbits)] `where `nbits` is for the mantissa.
    - `D` : a 64 bit floating point (`Double_t`)
-   - `d` : a 32 bit truncated floating point (`Double32_t`): 1 for the sign, 8 for the exponent and 23 for the mantissa. Can be customized with suffix `[min,max(,nbits)]`.
+   - `d` : a 24 bit floating point with truncated mantissa (`Double32_t`, stored as 3 bytes, occupies 8 bytes in memory): By default, in disk, 1 bit is used for the sign, 8 for the exponent and 15 for the mantissa. Can be customized with suffix `[min,max(,nbits)]` where `nbits` is for the mantissa.
```

Comment:
This does not seem to be accurate/clear.  The default is literally to store the value the same way we store C++ float (4 bytes resulting in the truncation of the both the double's mantissa and exponent).